plugins/pulp_rpm/plugins/migrations/0031_regenerate_repo_unit_counts.py

```diff
@@ -44,7 +44,7 @@ def migrate(*args, **kwargs):
     :type  kwargs: dict
     """
     db = connection.get_database()
-    repos_collection = db['repos']
-    for repo in repos_collection.find():
-        repo_id = repo['repo_id']
+    repos_collection = db['repos'].find({}, {'repo_id': 1})
+    repo_ids = [x['repo_id'] for x in repos_collection]
+    for repo_id in sorted(repo_ids):
```

Comment:
I don't know. It was in the proposed solution in the redmine issue. If we don't know what it's for, I can remove it.